cfgov/unprocessed/css/misc.less

```diff
@@ -166,14 +166,21 @@
     }
 }
 
+.float-media, .media_image {
```

Comment:
Break these class hooks onto individual lines to stick with existing code style.